tests/factories.py

```diff
@@ -20,7 +20,7 @@ class Meta:
     id = timeflake.random()  # noqa: A003
     user = factory.SubFactory("tests.factories.UserFactory")
     title = factory.Sequence(lambda n: f"Analysis Request {n}")
-    start_date = factory.Faker("date")
-    end_date = factory.Faker("date")
+    start_date = "2021-01-01"
```

Comment:
How come the date has been hard-coded to a single day?

Reply:
Because we need something valid as a default. The Faker versions would sometime produce a an end date before a start date, resulting in no outputs

It's not hardcoded, its just the default. You can do `AnalysisRequestFactory(start_date=...)` to alter it.

I guess I could use the current default [START_DATE](https://github.com/opensafely-core/interactive.opensafely.org/blob/main/interactive/models.py#L35)?

Reply:
Fair point about the end date being before the start date. I guess I was a bit worried about only querying one day. The actual application code is using START_DATE and END_DATE, so maybe the tests should use that too?